inventory_management_system_api/services/item.py

```diff
@@ -74,6 +81,10 @@ def create(self, item: ItemPostRequestSchema) -> ItemOut:
         except InvalidObjectIdError as exc:
             raise DatabaseIntegrityError(str(exc)) from exc
 
+        usage_statuses = [usage_status.value for usage_status in self._usage_status_repository.list()]
+        if item.usage_status not in usage_statuses:
+            raise MissingRecordError(f"No usage status found with name: {item.usage_status}")
```

Comment:
Do we need to check this because we are not doing this for units and to me they are both of equal importance? So perhaps we need to do the same check for units if this is actually needed. But the more important question is whether the API should require for `usage_status_id` to be supplied by the client rather than name/value?

Reply:
We did debate it ourselves, the reason units originally didn't have the check is because we had discussed it as being a placeholder to be edited later, but I do think it could be checked now although I prefer checking using the id rather than pulling the whole list and checking each value, so I feel they are one and the same issue (although we currently have less usage statuses). If we changed this to send back ids, we should do the same for units but as we don't have aggregations and its faster to just store the value, I don't see what the benefit would be to move to ids like that just yet.

Reply:
> the API should require for `usage_status_id` to be supplied by the client rather than name/value?

By this, I meant that the API expects the `usage_status_id` to be supplied by the client in the item `POST` and `PATCH` requests but the API returns the actual usage status name/value (and potentially `usage_status_id`) in the response. This way it would check if the usage status exists using the ID as I also don't like retrieving the whole list and checking each one, and would store both the `usage_status_id` and usage name/value in the database for each item. Storing the IDs means that we won't have to do any ID back population when we get aggregations working nicely in future.

Reply:
This is now addressed in https://github.com/ral-facilities/inventory-management-system-api/pull/248